From a93f1263bd59426604af36a4b7e9be22e80168ab Mon Sep 17 00:00:00 2001
From: Wong Vee Khee <vee.khee.wong@intel.com>
Date: Thu, 25 Jun 2020 13:12:31 +0800
Subject: [PATCH 01/11] SQUASHME: net: phy: fix 'int_name' buffer overflow
 issue found by Klocwork

This patch fixed variable 'int_name' buffer overflow issue found by
Klocwork.

Signed-off-by: Wong Vee Khee <vee.khee.wong@intel.com>
---
 drivers/net/phy/dwxpcs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/phy/dwxpcs.c b/drivers/net/phy/dwxpcs.c
index b59a886283ee..493907bfbc1a 100644
--- a/drivers/net/phy/dwxpcs.c
+++ b/drivers/net/phy/dwxpcs.c
@@ -103,7 +103,7 @@ struct dwxpcs_priv {
 	struct mdio_device *mdiodev;
 	struct pcs_stats stats;
 	struct dwxpcs_platform_data *pdata;
-	char int_name[IFNAMSIZ];
+	char int_name[IFNAMSIZ + 3];
 	unsigned long state;
 	struct workqueue_struct *int_wq;
 	struct work_struct an_task;
-- 
2.17.1

